### ISSUE TITLE
coco128

### ISSUE THREAD

**Issue by www516717402** [ultralytics/yolov5#3188]
## ❔Question
1. I get poor result when I train coco128 without pretrained model  and any data-augment. 128 images will fitness easy by yolov5s model.class loss is difficult to fit. The result as follows. Many object will recognize mistake to person class.

![image](https://user-images.githubusercontent.com/30460509/118460432-4857ed80-b72f-11eb-962e-0c653efd4d89.png)

**Comment by sezer-muhammed** [comment]
Hi,

If there is no mistake about classes or training, the model can detect objects in training images.
But for external images these results are normal. 128 images is very low for any yolov5 model.

If this image in already in training dataset, you may increase tranining time.

Or, try to change CUDA version. When I use 1660Ti I cannot train my model with CUDA 11.1.
When I use RTX3070 I cannot train my model with CUDA 10.2

I dont know why but if your mAP value does not increase, you may try change your cuda version.

**Comment by www516717402** [comment]
@sezer-muhammed Thank you for reply. 
1. I will check dataset again and increase trainning time by your advice.
2. Whether raise error when cuda not match?

**Comment by sezer-muhammed** [comment]
> @sezer-muhammed Thank you for reply.
> 
> 1. I will check dataset again and increase trainning time by your advice.
> 2. Whether raise error when cuda not match?

You are welcome, Yes.
If there is a CUDA problem in every epochs your mAP valuse will decrease.

**Comment by www516717402** [comment]
Great help. I will try. 
Thank you again.
